resources/lib/controller.py

```diff
@@ -676,10 +808,11 @@ def view_episodes(args, api: API):
             view.add_item(
                 args,
                 {
-                    "title": item["series_title"] + " #" + str(item["episode_number"]) + " - " + item["title"],
+                    "title": (str(item["season_number"]) + "x" if item["season_number"] else "") + utils.twoDigits(item["episode_number"]) + ". " + item["title"],
```

Comment:
I'm not yet sure about this and the same change in model.py, have to test it first. I, for one, like to have the series title to be prepended, at least in the watchlist.

Reply:
It probably needs 2 formats or a setting. Reading episode list from a season is a pain with the old format, especially when series title is long. But, you probably right for history or watchlist (it depends on your XBMC theme).

Reply:
I agree with not showing the season name on the episodes, as you already know which series/season you are looking at. But for the watchlist we should keep it. 

I'm not for adding another setting just for this, yet, we should keep it plain and simple for now.
It might be a good idea for the future to add an "Advanced settings" option to fine tune things and have a basic settings page for the starters with just minimal options.

Reply:
I'd add that to the continue watching menu as well. I just opened it and I'm totally lost what is what. :)

Reply:
An easier rule: keep the season name in episode title anywhere but in a season episode list.